osu.Game/Overlays/Toolbar/ToolbarMusicButton.cs

```diff
@@ -21,7 +21,7 @@ namespace osu.Game.Overlays.Toolbar
 {
     public partial class ToolbarMusicButton : ToolbarOverlayToggleButton
     {
-        private Circle volumeBar;
+        private Box volumeBar;
```

Comment:
Changed this to Box, because for some reason, the masking did not work with the Circle, might be a framework issue?

Reply:
Yes, see https://github.com/ppy/osu-framework/issues/2107.

Reply:
I presume this solution is good for now?

Reply:
Yes, using `Box` here is more correct as the corner radius is handled by the parent now.